docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM alpine:3.5
+FROM alpine:3.15
 
 COPY bin/pubsub /usr/local/bin/pubsub
 
 # Allows to verify certificates
-RUN apk update --no-cache && apk add --no-cache ca-certificates
+RUN apk update --no-cache && apk add --no-cache ca-certificates && apk upgrade
```

Comment:
`apk upgrade` in a container is considered bad practice because it can essentially double the size of the image. The original versions of files will still be in the base layer and this layer will have copies of all the updated files. One way around it, if needed, is to flatten the image using multi-stage builds, but it's probably best to just reference the latest base image.

Reply:
Sorry, I meant most recent tag, not `latest`.